packages/golden-layout/src/controls/DragProxy.js

```diff
@@ -1,3 +1,6 @@
+import $ from 'jquery';
+import utils from '../utils/index.js';
```

Comment:
Don't need `/index.js`
```suggestion
import utils from '../utils';
```